admin/release.py

```diff
@@ -559,23 +565,28 @@ def upload_rpms(scratch_directory, target_bucket, version, build_server):
         target_distro_suffix = ""
 
     operating_systems = [
-        {'distro': 'fedora', 'version': '20', 'arch': 'x86_64'},
-        {'distro': 'centos', 'version': '7', 'arch': 'x86_64'},
-        {'distro': 'ubuntu', 'version': '14.04', 'arch': 'amd64'},
+        {'distro': 'fedora', 'version': '20', 'arch': 'native'},
+        {'distro': 'centos', 'version': '7', 'arch': 'native'},
+        {'distro': 'ubuntu', 'version': '14.04', 'arch': 'native'},
+        {'distro': 'ubuntu', 'version': '15.04', 'arch': 'native'},
```

Comment:
It wasn't there and then @jongiddy suggested it in a review comment https://github.com/ClusterHQ/flocker/pull/1481#discussion_r31619597.